pkg/controller/druid/handler.go

```diff
@@ -174,7 +174,11 @@ func deployDruidCluster(sdk client.Client, m *v1alpha1.Druid) error {
 				if !done {
 					return err
 				}
+
 			}
+
+			checkCrashStatus(sdk, m)
+
```

Comment:
This should ideally be true only for sts with podmanagement policy orderready, since its only for orderready.

Reply:
@himanshug in case you get time, do review this one :)